validator/testcases/regex.py

```diff
@@ -2901,9 +2915,34 @@ def munge_filename(name):
                         "the `%s` preference branch" % branch)))
         for branch, reason in BANNED_PREF_BRANCHES))
 
-STRING_REGEXPS += PREF_REGEXPS
+# For tests in literal strings, add help text suggesting passing the
+# preference directly to reference getter functions.
+PREF_STRING_HELP = (
+    "If you are reading, but not writing, this preference, please consider "
+    "passing a string literal directly to `Preferences.get()` or "
+    "`nsIPrefBranch.get*Pref`.")
+
+
+def maybe_tuple(value):
+    if isinstance(value, tuple):
+        return value
+    return (value,)
```

Comment:
I'd rather see this in an `else:`. It makes it clearer that there are two separate cases, in my opinion.

```py
def to_tuple(value):
    if isinstance(value, tuple):
        return value
    else:
        return (value,)